src/chi/javascript/components/tooltip.js

```diff
@@ -92,12 +87,11 @@ class Tooltip extends Component {
   _createTooltip () {
     this._tooltipElem = document.createElement('div');
     this._tooltipElem.setAttribute('class', 'chi-tooltip');
-    this._tooltipElem.id = COMPONENT_TYPE +
-      Util.getData(this._elem, COMPONENT_TYPE);
+    this._tooltipElem.id = `chi-${COMPONENT_TYPE}-${this.componentCounterNo}`;
     this._tooltipContent = document.createElement('span');
     this._tooltipContent.innerText = this._elem.dataset.tooltip;
     this._tooltipElem.appendChild(this._tooltipContent);
-    this._elem.parentNode.appendChild(this._tooltipElem);
+    document.querySelector('body').appendChild(this._tooltipElem);
```

Comment:
Please, remove this element from the body when disposing.

Reply:
```
this._tooltipElem = null;
this._tooltipContent = null;
```
It is being removed when dispose is triggered

Reply:
You are clearing local variables, but it doesn't mean the element is removed from the body. It is about references. You have deleted the local reference to the element, but the element is still there.